extension/src/completion/autocompletionProvider.ts

```diff
@@ -114,9 +114,14 @@ function getMatchingWord(document: vscode.TextDocument, position: vscode.Positio
     return [word, inputIsUpper];
 }
 
-function getLispAndDclCompletions(document: vscode.TextDocument, word: string, isupper: boolean): vscode.CompletionItem[] {
+export function getLispAndDclCompletions(document: vscode.TextDocument, word: string, isupper: boolean): vscode.CompletionItem[] {
     let currentLSPDoc = document.fileName;
     let ext = currentLSPDoc.substring(currentLSPDoc.length - 4, currentLSPDoc.length).toUpperCase();
+    if(internalLispFuncs == [] || internalDclKeys == [] || allCmdsAndSysvars == [] || winOnlyListFuncPrefix == []){
+        console.log('Reload the resource since the value are empty');
+        
+        loadAllResources();
```

Comment:
`loadAllResources()` is supposed to happen once on extension load, maybe it needs to stay where it is (in the extension.ts) and just be turned into an awaited function? IE, you may have a unfulfilled promise issue.

I just know it can't be here, this function gets called constantly because it is activated by user typing.

Reply:
The problem here is that on Github CI virtual enviornment I found the extension is not actually activated so I tried to load it before using it.

Reply:
@nigma456 , I didn't see your comment and edited mine. So, making a reply.
looking at this further, `resources.ts` has a lot of FS ReadFile() going on that may need to be turned into [ReadFileSync](https://nodejs.dev/learn/reading-files-with-nodejs).